driver/tests/amr/amr_dx1.yaml

```diff
@@ -0,0 +1,50 @@
+physics:
+  flow:
+    mode: swe
+
+numerics:
+  spatial: fv
+  temporal: euler
+  riemann: roe
+
+logging:
+  level: debug
+
+time:
+  final_time: 0.05
+  unit: seconds
+  time_step : 0.005
+  coupling_interval: 0.01
+
+#output:
+#  format: xdmf
+#  step_interval: 100
+
+grid:
+  file: mms_triangles_dx1.exo
```

Comment:
worth making an AMR-specific copy of this mesh to keep things separate?

Reply:
For now, let's reuse the MMS mesh. In the next major AMR PR, we would add a realistic mesh for AMR.